static/styles/javascript/scripts.js

```diff
@@ -0,0 +1,98 @@
+$(document).ready(function() {
+
+    var writeMessage;
+    // building message element and appending to the list of all messages
+    writeMessage = function(message) {
```

Comment:
Is there a benefit to having these function declarations rather than statements (e.g. `function writeMessage(message){}`)?

Reply:
No. Fixed.